NEWS.md

```diff
@@ -14,6 +14,10 @@
 * Support for customizing Windows targets on darwin/arm64 (#1260)
 * Support for customizing Windows targets on WSL without Wine installed (#1260)
 
+### Added
+
+* `extendHelperInfo` option to allow extending helper app `Info.plist` files
+
```

Comment:
Needs to go above the 15.3.0 release notes (also reference this PR while you're at it)